sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -142,8 +142,10 @@ export class BatchingReceiver extends MessageReceiver {
 
       // Action to be performed on the "receiver_drained" event.
       onReceiveDrain = (context: EventContext) => {
-        this._receiver!.removeListener(ReceiverEvents.receiverDrained, onReceiveDrain);
-        this._receiver!.drain = false;
+        if (this._receiver) {
```

Comment:
I recall seeing this recent change to make receiver nullable. Was there a reason why ?

Reply:
I am not aware of what exactly are you referring to. `this._reciever` was always nullable